src/main/java/com/wepay/zktools/zookeeper/internal/ZooKeeperClientImpl.java

```diff
@@ -54,7 +57,25 @@
 
     private volatile boolean running = true;
 
+    /**
+     * Constructor to create a ZooKeeperClientImpl instance.
+     * @param connectString comma separated host:port pairs, each corresponding to a zk server.
```

Comment:
nit: its just one zk server in host:port format. Not multiple of them.

Reply:
> nit: its just one zk server in host:port format. Not multiple of them.

Actually in [apache.ZooKeeper class](https://github.com/apache/zookeeper/blob/master/zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeper.java#L797) it's saying *comma separated host:port pairs, each corresponding to a zk server. e.g. "127.0.0.1:3000,127.0.0.1:3001,127.0.0.1:3002"*. Does zktools explicitly only support one host:port?

Reply:
aaahh you are right. Didn't know about this. I take back my statement.